llvm/test/CodeGen/AMDGPU/si-annotate-dbg-info.ll

```diff
@@ -0,0 +1,163 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -mtriple=amdgcn-- -S -structurizecfg -si-annotate-control-flow %s | FileCheck -check-prefix=OPT %s
```

Comment:
I expected this to use new PM, but it looks like is-annotate-control-flow hasn't been ported yet